Block/Adminhtml/Group/Edit/Form.php

```diff
@@ -0,0 +1,227 @@
+                $groupId
+            );
+        }
+
+        $fieldset = $form->addFieldset('base_fieldset', ['legend' => __('Group Information')]);
+
+        $validateClass = sprintf(
+            'required-entry validate-length maximum-length-%d',
+            \Magento\Customer\Model\GroupManagement::GROUP_CODE_MAX_LENGTH
+        );
+        $name = $fieldset->addField(
+            'customer_group_code',
+            'text',
+            [
+                'name' => 'code',
+                'label' => __('Group Name'),
+                'title' => __('Group Name'),
+                'note' => __(
+                    'Maximum length must be less then %1 characters.',
+                    \Magento\Customer\Model\GroupManagement::GROUP_CODE_MAX_LENGTH
+                ),
+                'class' => $validateClass,
+                'required' => true
+            ]
+        );
+
+        if ($customerGroup->getId() == 0 && $customerGroup->getCode()) {
+            $name->setDisabled(true);
+        }
+
+        $fieldset->addField(
+            'tax_class_id',
+            'select',
+            [
+                'name' => 'tax_class',
+                'label' => __('Tax Class'),
+                'title' => __('Tax Class'),
+                'class' => 'required-entry',
+                'required' => true,
+                'values' => $this->_taxCustomer->toOptionArray(),
+            ]
+        );
+        $fieldset->addField(
+            'enable_balance_payments',
+            'checkbox',
+            [
+                'name' => 'enable_balance_payments',
+                'label' => __('Enable Balance payments'),
+                'title' => __('Enable Balance payments'),
+                'class' => '',
+                'value' => 1,
+                'checked' => $this->getEnableBalancePay($groupId),
+            ]
+        );
+
+        $fieldset->addField(
+            'customer_group_excluded_website_ids',
+            'multiselect',
+            [
+                'name' => 'customer_group_excluded_websites',
+                'label' => __('Excluded Website(s)'),
+                'title' => __('Excluded Website(s)'),
+                'required' => false,
+                'can_be_empty' => true,
+                'values' => $this->systemStore->getWebsiteValuesForForm(),
+                'note' => __('Select websites you want to exclude from this customer group.')
+            ]
+        );
+
+        if ($customerGroup->getId() !== null) {
+            // If edit add id
+            $form->addField('id', 'hidden', ['name' => 'id', 'value' => $customerGroup->getId()]);
+        }
+
+        if ($this->_backendSession->getCustomerGroupData()) {
+            $form->addValues($this->_backendSession->getCustomerGroupData());
+            $this->_backendSession->setCustomerGroupData(null);
+        } else {
+            // TODO: need to figure out how the DATA can work with forms
```

Comment:
Can you resolve this?

Reply:
@ynaxonBalance 
This is core Magento code as we have overwritten the file.